src/transactions/stake.js

```diff
@@ -1,10 +1,17 @@
 import algosdk from "algosdk";
 import { ids } from "./ids";
-import { setLoadingStage, microGARD, getMicroGardBalance } from "./lib"
+import { setLoadingStage, microGARD, getMicroGardBalance, getAppField, cdpInterest } from "./lib"
 import { accountInfo, getParams, signGroup, sendTxn } from "../wallets/wallets";
 
 const enc = new TextEncoder();
 
+export async function getStakingAPY(pool) {
+  const expectedBonus = 5000 * 52 * 1000000
+  const nltvlpromise = getAppField(ids.app.gard_staking, pool)
+  const gardIssued = await getAppField(ids.app.validator, "GARD_ISSUED")
+  return 100 * (cdpInterest * gardIssued + expectedBonus) / (await nltvlpromise)
```

Comment:
This is overcounting because the NL pool won't receive 100% of protocol interest. Nor will the NL pool receive 100% of the staking bonus once people stake in other pools.

Reply:
Adding an (initially hardcoded) variable for this